dbt/include/trino/macros/materializations/incremental.sql

```diff
@@ -69,6 +119,7 @@
 
   {#-- Validate early so we don't run SQL if the strategy is invalid --#}
   {%- set strategy = validate_get_incremental_strategy(config) -%}
+  {% set incremental_predicates = config.get('incremental_predicates', none) %}
```

Comment:
@hovaesco  if we don't use the `incremental_predicates` there's no reason to read them. this is probably a left-over.

Reply:
Done